### PR TITLE
[GOVCMSD8-453] Add minisite 8.x-1.3

### PR DESCRIPTION
**Pull request by pandaskii**
This module provides the ability to upload static 'minisites' to a Drupal website and maintain the minisite's look and feel.

Please note this module does not actually import the minisite pages as Drupal nodes.

The minisite module provides an additional custom field which follows a user story that "Site editor can upload a zipped archive attached to a node, and automatically extract this into the files directory. When the node is deleted, it should clean up all files uploaded. If a zip file contains any not-allowed extensions, the entire upload must be aborted, and no files saved to the file system."

upload and extract minisite archives
file blacklisting
file whitelisting
permission control
Suggested approach was to upload a zipped archive attached to a node, and automatically extract this into the files directory. When the node is deleted, it should clean up all files uploaded. If a zip file contains any not-allowed extensions, the entire upload must be aborted, and no files saved to the file system.

https://www.drupal.org/project/minisite/releases/8.x-1.0

### CODE REVIEW

**Review by fubarhouse — DISMISSED** [review]

**Review by simesy — DISMISSED** [review]

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]
Please use the newest version `1.1`

**Review by AlexSkrypnyk — DISMISSED** [review]

**Comment by mingsong-hu** [comment]
The 8.x-1.2 version is released.

**Comment by mingsong-hu** [comment]
Patch to fix the D9 deprecated code issue for function file_uri_target()

**Comment by AlexSkrypnyk** [comment]
@AMDS 
Thanks for providing the patch, but it should have been provided as a PR on GitHub version of the project (https://github.com/salsadigitalauorg/minisite) to make sure that it does not brake the functionality of the module.

I suggest to delay merging of this PR until the patch can be applied to Minisite in GitHub and a new version is released.

I'm keen to release `1.3` with this patch if someone can please open a PR in https://github.com/salsadigitalauorg/minisite.

**Comment by AlexSkrypnyk** [comment]
Blocked this until `1.3` is released with D9 patch applied.
Can be tracked here https://github.com/salsadigitalauorg/minisite/pull/25

**Comment by AlexSkrypnyk** [comment]
Version 1.3 released with applied patch. 
https://www.drupal.org/project/minisite/releases/8.x-1.3
@AMDS could you please update this PR to include `1.3`.

File: composer.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
         "drupal/menu_block": "1.5",
         "drupal/menu_trail_by_path": "1.1",
         "drupal/metatag": "1.9",
+        "drupal/minisite": "1.2",
```
Comment (AlexSkrypnyk) [inline composer.json:78]: please change to 1.3

Comment (AlexSkrypnyk) [inline composer.json]: please update to 1.3

File: composer.json [hunk 2]
```diff
@@ -152,6 +153,9 @@
             "drupal/metatag": {
                 "Unset link rel: shortlink, edit-form, version-history and revision, and meta name: Generator, MobileOptimized, HandheldFriendly, viewport from <head> - https://www.drupal.org/project/metatag/issues/2735195#comment-12991008": "https://www.drupal.org/files/issues/2019-02-28/metatag-unset-2735195-42.patch"
             },
+            "drupal/minisite": {
```
Comment (AlexSkrypnyk) [inline composer.json]: please remove this as this patch is already applied to `1.3`

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]

**Review by AlexSkrypnyk — APPROVED** [review]

**Comment by AlexSkrypnyk** [comment]
Approved

**Pull request merged**